tss-esapi/src/structures/names/name.rs

```diff
@@ -35,7 +35,7 @@ impl TryFrom<Vec<u8>> for Name {
         }
         let size = bytes.len() as u16;
         let mut name = [0; Name::MAX_SIZE];
-        name.copy_from_slice(&bytes);
+        name[..bytes.len()].copy_from_slice(&bytes);
```

Comment:
why not `..size`?

Reply:
Because `size` is `u16`, while this number has to be `usize`. Could do a cast, but it ends up being just as bad